### PR TITLE
added new moveType

### PR DESCRIPTION
**Pull request by andrewjmeier**
This is a new move type that works for games like connect 4 and tic tac toe. Unlike moveTypeManual, moveTypePlaceToken does not have a token click to start the move. The user simply picks a spot on the board and clicks a tile and then a token is placed there. 

This changes the way that tokens are drawn (because we don't draw them all at the beginning of the game) and the way the game determines if a valid move has been made.

### CODE REVIEW

File: tabletop/core/game.js [hunk 1]
```diff
@@ -151,30 +151,32 @@ Game.prototype.setProposedMoveToken = function(token) {
...
 
 Game.prototype.isValidMove = function(token, oldSpace, newSpace) { 
-  console.warn("isValidMove should be implemented by subclass!");
+  console.log("Warning: you should overwrite isValidMove(token, oldSpace, newSpace)");
   return true;
 };
 
 Game.prototype.playerDidWin = function(player) {
-  console.warn("playerDidWin should be implemented by subclass!");
+  console.log("Warning: you should overwrite playerDidWin(player)");
```
Comment (konnorbeard) [inline tabletop/core/game.js]: What's the reason for this change?

File: tabletop/core/game.js [hunk 2]
```diff
@@ -151,30 +151,32 @@ Game.prototype.setProposedMoveToken = function(token) {
...
   return false;
 };
 
-Game.prototype.executeMove = function(player) {
-  throw new Error('must be implemented by subclass!');
-};
-
```
Comment (konnorbeard) [inline tabletop/core/game.js]: Why don't we want this?

File: tabletop/core/game.js (hunk 1 above)
Comment (andrewjmeier) [inline tabletop/core/game.js]: Oh I didn't do that intentionally

File: tabletop/core/game.js (hunk 2 above)
Comment (andrewjmeier) [inline tabletop/core/game.js]: Another thing that just got missed in some copy paste

**Comment by konnorbeard** [comment]
lgtm
